test/commonSvg.test.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-undef */
+import React from 'react'
+// import renderer from 'react-test-renderer'
+import { EOS_10K } from '../lib'
+import { render, cleanup } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+
+afterEach(cleanup)
+
+it('Filled SVG Size prop test', () => {
```

Comment:
These tests are for default components so we can change the test name to `Test default component size prop`. Similar for other tests.

Reply:
Thanks for pointing this out, I have changed the naming to `default filled component` as the default component renders the filled variant when nothing is supplied to the theme prop.

I have also added new test cases for the `default outlined component`, in which I have supplied "outlined" as a prop to Theme